src/preload.ts

```diff
@@ -0,0 +1,71 @@
+import { contextBridge } from 'electron'
```

Comment:
Nice! Hadn't seen this API before, it makes complete sense to use it instead of node integration 😄